tools/test-proxy/docker/dev_certificate/apply-dev-cert.sh

```diff
@@ -28,4 +28,4 @@ fi
 $SUDO cp $CRTFILE "/usr/local/share/ca-certificates"
 $SUDO update-ca-certificates
 
-dotnet dev-certs https --clean --import $PFXFILE -p ""
+dotnet dev-certs https --clean --import $PFXFILE -p "password"
```

Comment:
Is that a real password?

Reply:
This is a test certificate. Yes.

Reply:
To be entirely clear, this is _specifically_  an ASP.NET Development certificate, generated for common name `localhost`. It's not really an abusable resource, as `localhost` certificates are strictly not really secure. No browser will ever allow spoofing using this certificate.